docs/modules/publisher-directmethods.md

```diff
@@ -0,0 +1,328 @@
+[Home](../../readme.md)
+
+# Configuration via IoT Hub Direct methods
+
+OPC Publisher v2.8.2 implements the following [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application (anywhere in the world) leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks).
+
+There are some direct methods which are inherited from 2.5.x in addition to new ones.
+
+Inherited direct methods from 2.5.x:
+
+- PublishNodes_V1
+- UnpublishNodes_V1
+- UnpublishAllNodes_V1
+- GetConfiguredEndpoints_V1
+- GetConfiguredNodesOnEndpoints_V1
+- GetDiagnosticInfo_V1
+
+New direct methods:
+
+-  AddOrUpdateEndpoints_V1
+
+Note: `_V1` suffix is not required.
```

Comment:
I think _V1 is required in 2.8.2. can you validate and rephrase this?

Reply:
It should be possible to keep the same "client" working against 2.5.  I think _V1 is not required, but if we ever did a _V2 it would default to that.  Let's make sure.

Reply:
I verified and _V1 is indeed required. This would actually be a change using 2.8.2 as _V1 is not required 2.5.x. I will rephrase this to be more evident.

Reply:
Based on the discussion, _V1 should not be required. Made the changes